From 91649556e1bef095c2a79ace0cfa5572aa503178 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 10 Jul 2017 11:06:51 +0800
Subject: [PATCH 2098/5242] MLK-15932-2 video: dpu: Remove the prefix 'lb_'
 for lb_pixengcfg_clken_t

commit  46b96dfc07fa6e33afea56c946f71366faa61402 from
https://source.codeaurora.org/external/imx/linux-imx.git

There are several DPU units which have the same clock enable control bits
in their Dynamic registers, e.g., HScaler, VScaler, Rop, Fliter and Matrix,
etc.  So, let's remove the prefix 'lb_' from the enumerator name of
lb_pixengcfg_clk_t so that it can be a little bit generic.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-layerblend.c |    2 +-
 include/video/dpu.h                  |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-layerblend.c b/drivers/gpu/imx/dpu/dpu-layerblend.c
index ef90ee6..c7c42f3 100644
--- a/drivers/gpu/imx/dpu/dpu-layerblend.c
+++ b/drivers/gpu/imx/dpu/dpu-layerblend.c
@@ -165,7 +165,7 @@ void layerblend_pixengcfg_dynamic_sec_sel(struct dpu_layerblend *lb,
 EXPORT_SYMBOL_GPL(layerblend_pixengcfg_dynamic_sec_sel);
 
 void layerblend_pixengcfg_clken(struct dpu_layerblend *lb,
-				lb_pixengcfg_clken_t clken)
+				pixengcfg_clken_t clken)
 {
 	u32 val;
 
diff --git a/include/video/dpu.h b/include/video/dpu.h
index 4418cc6..2dd96d5 100644
--- a/include/video/dpu.h
+++ b/include/video/dpu.h
@@ -350,7 +350,7 @@ enum dpu_irq {
 	CLKEN__DISABLE = 0x0,
 	CLKEN__AUTOMATIC = 0x1,
 	CLKEN__FULL = 0x3,
-} lb_pixengcfg_clken_t;
+} pixengcfg_clken_t;
 
 int dpu_map_irq(struct dpu_soc *dpu, int irq);
 
@@ -472,7 +472,7 @@ int layerblend_pixengcfg_dynamic_prim_sel(struct dpu_layerblend *lb,
 void layerblend_pixengcfg_dynamic_sec_sel(struct dpu_layerblend *lb,
 					  lb_sec_sel_t sec);
 void layerblend_pixengcfg_clken(struct dpu_layerblend *lb,
-				lb_pixengcfg_clken_t clken);
+				pixengcfg_clken_t clken);
 void layerblend_shden(struct dpu_layerblend *lb, bool enable);
 void layerblend_shdtoksel(struct dpu_layerblend *lb, lb_shadow_sel_t sel);
 void layerblend_shdldsel(struct dpu_layerblend *lb, lb_shadow_sel_t sel);
-- 
1.7.9.5

